### DISCUSSION TITLE
Does PrimeHub / InventorHub have a better IMU module than TechnicHub?

### DISCUSSION THREAD

**Discussion by Invenisso** [pybricks/support#1768] [Q&A] [answered]
The official Lego specification says that TechnicHub only has a tilt sensor - while PrimeHub has a Six-axis Gyro Sensor: Three-axis accelerometer • Three-axis gyroscope • Able to report: • Gyroscope mode (three-axis) • Accelerometer/tilt mode (three -axis) • Gestures as tap, free fall, and shake. 

However, the API provided by Pybricks does not differ regarding IMU between these hubs (as far as I know).
So perhaps the Pybricks IMU features available on TechnicHub are somehow emulated or interpolated from what inferior hardware provides?

**Comment by dlech** [comment, marked as answer]
The BOOST Move hub only has 3-axis accelerometer, but all of the other hubs have the same 6-axis IMU. LEGO documentation about sensors doesn't tend to use technically precise language, so when the say "tilt" sensor, it can mean a lot of different things.
